www/app/app.js

```diff
@@ -135,6 +114,71 @@ define(['angularAMD', 'app.routes', 'app.constants', 'app.permissions', 'domotic
         }
     });
 
+	/*
+		The notifyBrowser service sets and gets the browser notifications coming on from the domoticz websocket.
+		In livesocket.js the notifications are pushed to the message queue by using the notify() method.
+		Each notification times out in 2 seconds.
+		The alertarea directive reads the messagequeue using the messages() method.
+	*/
+	app.service('notifyBrowser', [ '$timeout', function ($timeout) {
+		var messagequeue = [];
+		return {
+			notify: function (title, body) {
+				var item = { "title": title, "body": body, "time": new Date() };
+				messagequeue.push(item);
+				$timeout(function () {
+					var index = messagequeue.indexOf(item);
+					if (index >= 0) {
+						messagequeue.splice(index, 1);
+					}
+				}, 2000);
+				if (typeof Notification == "undefined") {
+					console.log("Notification: " + title + ": " + body);
+					console.log('Desktop notifications not available in your browser. Try Chromium.');
+					return;
+				}
+
+				if (Notification.permission !== "granted")
+					Notification.requestPermission();
+				else {
+					var notification = new Notification(title, {
+						//icon: 'http://cdn.sstatic.net/stackexchange/img/logos/so/so-icon.png',
+						body: body,
+					});
+
+					notification.onclick = function () {
```

Comment:
Should the onclick hide the notification or something else ?
Now it opens a window to a stackoverflow page ;)